python/main.py

```diff
@@ -19,34 +23,44 @@
     allow_headers=["*"],
 )
 
+def dict_factory(cursor, row):
+    d = {}
+    for idx, col in enumerate(cursor.description):
+        if col[0] == 'id':
+            continue
+        d[col[0]] = row[idx]
+
+    return d
+
 @app.get("/")
 def root():
     return {"message": "Hello, world!"}
 
 @app.post("/items")
 def add_item(name: str = Form(...), category: str = Form(...)):
-    logger.info(f"Receive item: {name}")
-    logger.info(f"Receive item category: {category}")
+    logger.info(f"Receive item: {name} in {category}")
+
+    item = (name, category)
 
-    with open('items.json', 'r+') as jsonfile:
-        items = json.load(jsonfile)
-        items['items'].append({
-            'name': name,
-            'category': category
-        })
+    conn = sqlite3.connect(DBPATH)
+    c = conn.cursor()
+    c.execute("INSERT INTO items VALUES (NULL,?,?)", item)
```

Comment:
You may choose which columns to insert data by adding parentheses with column names in it:
`INSERT INTO items(column_name_1, column_name_2, ... etc) VALUES (?, ?, ...)`
This way you do not have to use "NULL" in VALUES.

Reply:
Thank you for the advice.